src/views/profile/index.vue

```diff
@@ -337,7 +346,8 @@ export default {
             } else {
               return '-'
             }
-          }
+          },
+          has: !store.getters.publicSettings['PRIVACY_MODE']
         },
         {
           value: this.object.groups?.map(item => item.name).join(' ｜ '),
```

Comment:
None of these code snippets have been identified with irregularities, issues, or optimizations to suggest at the moment given the knowledge cutoff provided (2021-09-01). This is because they seem fine from my current perspective based on information I was trained up until that point.

Please inform me if you would like an evaluation after September 2021.